R/clustering.R

```diff
@@ -82,6 +85,13 @@ FindClusters.default <- function(
             edge.file.name = edge.file.name
           )
         } else if (algorithm == 4) {
+          if (method == "matrix" || method == "igraph"){
+            if (method == "igraph"){
+            object <- graph_from_adjacency_matrix(object)
+            }
+          } else {
+            stop("method must be 'matrix' or 'igraph'")
+          }
```

Comment:
Why are we doing these checks if we're parallelizing the clustering but not otherwise? Also, wouldn't this be better implemented in `RunLeiden`?

Reply:
Whoops, forgot to refresh the files changed list